config/crd/bases/cassandra.datastax.com_cassandradatacenters.yaml

```diff
@@ -140,6 +140,61 @@ spec:
                   size, then all nodes in the rack will get updated.
                 format: int32
                 type: integer
+              cdc:
+                description: CDC allows configuration of the change data capture agent
```

Comment:
We should probably mention here that this isn't usable for generic CDC usage, only Datastax Pulsar integration.

Reply:
Pulsar isn't limited a DataStax proprietary product... I imagine in future we'll see connectors which might allow output to Kafka et al. via some "Pulsar on Kafka" type technology.

In saying that, if you provide me your preferred phrasing I'm probably happy to document as you like.

Reply:
@burmanm - waiting on a response on this one too.

Reply:
This only configures the Datastax CDC client (not Cassandra's internal CDC - these features do not allow setting any properties to support that feature). And as you said in earlier comments on structure, this is just for the Datastax agent which only supports Pulsar.

Reply:
I don't know about this. Cassandra doesn't really have internal CDC. It has a hook which juggles some files containing the mutations into a different folder. The more I think about it, the less I agree with this suggestion...